src/language.cpp

```diff
@@ -233,6 +257,7 @@ void StringParameters::Clear()
 {
 	this->parms.clear();
 	this->set_mode = true;
+	this->pluralize_amount = 1;
```

Comment:
This is the number of plural forms?  "count" is much better here then, or "number_plurals" (a bit short though, improvement is possible).

"amount" is very generic notion, beyond "it's a number" it doesn't say much.

Reply:
If it is the first plural form, wouldn't be `0` a more logical choice?